README.md

```diff
@@ -1,6 +1,7 @@
 # LBH Tenancy Information API
 
 Tenancy Information API returns information about tenancies held
```

Comment:
Could we make this clearer too? Maybe use what's in swagger?

Reply:
Swagger one is not great:
API for accessing tenancy information